provider/kubernetes.go

```diff
@@ -170,10 +170,16 @@ func (provider *Kubernetes) loadIngresses(k8sClient k8s.Client) (*types.Configur
 					log.Errorf("Error retrieving services %s", pa.Backend.ServiceName)
 				}
 				for _, service := range services {
-					var protocol string
+					protocol := "http"
 					for _, port := range service.Spec.Ports {
 						if port.Port == pa.Backend.ServicePort.IntValue() {
-							protocol = port.Name
+							if len(port.Name) == 0 {
+								if port.Port == 443 {
+									protocol = "https"
+								}
+							} else {
+								protocol = port.Name
```

Comment:
Yeah, I know, but I don't know how to do it otherwise...
By default it's http, https with 443. And you can override it with name.
Do I keep it like that?